.github/workflows/pr-comments.yaml

```diff
@@ -0,0 +1,57 @@
+name: pull request comment
+on:
+  issue_comment:
+    types: [created, edited, deleted]
+env:
+  GH_USER: "github-actions[bot]"
+  GH_EMAIL: "<41898282+github-actions[bot]@users.noreply.github.com>"
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Does this not run into the [issue with the token not triggering other workflows](https://docs.github.com/en/actions/security-guides/automatic-token-authentication#using-the-github_token-in-a-workflow)?

Reply:
Do you think it's not going to work against forks?

Reply:
Or are you worried that the commit coming from the action will not retrigger CI?

Reply:
The latter

Reply:
Hmm maybe. That's something we can fix later if that's the case I'll add a PAT like you've done for charts no?

Reply:
I suppose, it's maybe ok ish since circleci has our CI. I'm not sure it's fixable since forks don't run with our secrets but we can try it out like this, sure.